PATTERNS.md

```diff
@@ -1,6 +1,4 @@
-
-Patterns
-========
+# Patterns
```

Comment:
Do we need all these formatting changes in this PR? :) I would remove if unnecessary, its cluttering things up imo

Reply:
will make another pr

Reply:
https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint was used and automatically formatted it for me on save